nf_core/pipeline-template/.github/workflows/stale-tagged.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     steps:
       - uses: actions/stale@v7
         with:
-          stale-issue-message: "This issue is stale because it has been inactive for more than 30 days. More information is required. Remove stale label or comment or this will be closed in 20 days."
-          stale-pr-message: "This PR is stale because it has been open more than 45 days with no activity. Remove stale label or comment if it is still useful. In any case a PR will be automatically closed."
+          stale-issue-message: "This issue is stale because a user tagged it for stale. It has been inactive for more than 30 days. More information is required. Remove stale label or comment or this will be closed in 20 days."
+          stale-pr-message: "This PR is stale because a user tagged it for stale. It has been open more than 45 days with no activity. Remove stale label or comment if it is still useful. In any case a PR will be automatically closed."
```

Comment:
```suggestion
 stale-issue-message: "This issue has been tagged as stale by an nf-core contributor. Remove stale label or add a comment otherwise this issue will be closed in 20 days."
          stale-pr-message: "This PR has been tagged as stale by an nf-core contributor. Remove stale label or add a comment if it is still useful. Otherwise, this PR will be automatically closed."
```